cutter doesn't build with radare2 3.2.0 if "sys_magic=true" is selected

Hi,

I tried to compile latest cutter 1.7.3 on Mageia Cauldron but as reported here https://github.com/radare/radare2/issues/12768, cutter doesn't build if radare2 3.2.0 submodule is compiled with system magic (like "sys_magic=true" meson option), I get:

> moc_ColorSchemeFileSaver.o moc_ColorSchemePrefWidget.o moc_EditFunctionDialog.o moc_CutterTreeView.o moc_ComboQuickFilterView.o moc_HexdumpRangeDialog.o   -lr_core -lr_fs -lr_debug -lr_hash -lr_anal -lr_search -lr_egg -lr_crypto -lr_config -lr_bin -lr_io -lr_asm -lr_parse -lr_syscall -lr_reg -lr_lang -lr_cons -lr_flag -lr_socket -lr_bp -lr_util -lpython3.7m -lQt5Svg -lQt5Widgets -lQt5Gui -lQt5Network -lQt5Core -lGL -lpthread   
/usr/bin/ld: Cutter.o:(.data+0x74): undefined reference to `r_magic_version'
collect2: error: ld returned 1 exit status
make: *** [Makefile:727: Cutter] Error 1


For upstream radare2 this come from cutter who try to use radare2 magic library instead of system one.

[build.0.20190113133217.log](https://github.com/radareorg/cutter/files/2754470/build.0.20190113133217.log)

Regards,
David

--- a/src/Cutter.cpp
+++ b/src/Cutter.cpp
@@ -2265,7 +2265,9 @@ QString CutterCore::getVersionInformation()
         { "r_hash", &r_hash_version },
         { "r_fs", &r_fs_version },
         { "r_io", &r_io_version },
+#if !USE_LIB_MAGIC        
         { "r_magic", &r_magic_version },
+#endif
         { "r_parse", &r_parse_version },
         { "r_reg", &r_reg_version },
         { "r_sign", &r_sign_version },

